### DISCUSSION TITLE
what is github?

### DISCUSSION THREAD

**Discussion by aqsajamil** [community/community#21707] [New to GitHub]
<p>what is github?  why is it so imp to know</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>GitHub, at its core, is a <a href="https://github.com/about" rel="nofollow noopener">software code hosting and collaboration tool</a> with a community of over 28 million people (as of June of 2018) using it to build software, share information, collaborate on projects, offer support, and much more. In addition, we (GitHub, the company) create and support development tools such as <a href="https://atom.io" rel="nofollow noopener">Atom</a>, <a href="https://electronjs.org/" rel="nofollow noopener">Electron</a>, and <a href="https://desktop.github.com" rel="nofollow noopener">Desktop</a>. As for why it is important to know about, that would depend on where you’re coming from and what you’re expecting to be doing with it. Can you tell us a little bit about yourself and how you heard about GitHub?</p>

**Comment by lisa875** [comment]
<p>Hey lee,<br>
Great! I have got to know some useful Ideas. I really like the way you have explained Each and everything in Detail.<br>
Thank you for sharing it looking forward to reading more info from you.</p>

**Comment by lee-dohm** [comment]
<p>Thanks for the kind words <a class="mention" href="/lisa875">@lisa875</a> 🙇‍♂️ I’m glad to hear from you.</p>
<p>You may be interested in my <a href="/t5/Studies-in-Community/bg-p/StudiesinCommunity">Studies in Community series</a> where I explore various aspects of how to keep an online community safe and welcoming.</p>
